pallets/gear/src/benchmarking/mod.rs

```diff
@@ -872,6 +872,28 @@ benchmarks! {
         verify_process(res.unwrap());
     }
 
+    gr_error {
+        let r in 0 .. API_BENCHMARK_BATCHES;
+        let mut res = None;
+        let exec = Benches::<T>::gr_error(r)?;
+    }: {
+        res.replace(run_process(exec));
+    }
+    verify {
+        verify_process(res.unwrap());
+    }
+
+    gr_error_per_kb {
+        let n in 0 .. T::Schedule::get().limits.payload_len / 1024;
```

Comment:
```suggestion
        let n in 0 .. / view comment /;
```
I'd somehow go with MaxEncodedLen of the error we use at backend